project/plugins.sbt

```diff
@@ -8,7 +13,7 @@ addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "0.2.2")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.3")
 
-resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
+addSbtPlugin("com.eed3si9n" % "sbt-doge" % "0.1.5")
```

Comment:
from: https://github.com/sbt/sbt-doge I can see:  `This plugin is retired`, better to switch to sbt 1 instead of integrating an outdated plugin

Reply:
removed